generated/nifake/tests/test_session.py

```diff
@@ -539,9 +699,23 @@ def test_set_attribute_channel(self):
             session['0-24'].read_write_double = test_number
             self.patched_library.niFake_SetAttributeViReal64.assert_called_once_with(SESSION_NUM_FOR_TEST, b'0-24', attribute_id, test_number)
 
-    # TODO(marcoskirsch)
-    # def test_get_attribute_int64(self):
-    # def test_set_attribute_int64(self):
+    def test_get_attribute_int64(self):
+        self.patched_library.niFake_GetAttributeViInt64.side_effect = self.side_effects_helper.niFake_GetAttributeViInt64
+        attribute_id = 1000006
+        test_number = 3
```

Comment:
Use a number big enough that if 32 bits is used, it would be wrong.

Reply:
Fixed